packages/lib/src/interactions/SelectionTool.tsx

```diff
@@ -16,10 +18,10 @@ import { boundPointToFOV, getModifierKeyArray } from './utils';
 
 interface Props extends CommonInteractionProps {
   id?: string;
-  onSelectionStart?: () => void;
-  onSelectionChange?: (points: Selection) => void;
-  onSelectionEnd?: (points: Selection) => void;
-  children: (points: Selection) => ReactNode;
+  onSelectionStart?: (evt: CanvasEvent<PointerEvent>) => void;
```

Comment:
I'm now passing the start event through to the callback. No reason not to, really.